docs/repositories-configure/adjusting-quality-settings.md

```diff
@@ -1,45 +1,33 @@
 # Adjusting quality settings
 
-The **Quality settings** give you control over the status of analyzed commits and pull requests, as well as the quality level thresholds in your repository dashboard. This feature also allows you to turn off metrics like duplication or complexity.
+The **Quality settings** allow you to configure quality gates for your pull requests and commits, and define quality goals that help you monitor the progress of the code quality in your repository dashboard.
 
-To access the quality settings, open your repository **Settings**, tab **Quality Settings**. On this page there are three sections where you can set specific values at the commit, pull request, and repository levels. Click **Edit settings** to edit the settings on each section.
+To access the quality settings, open your repository **Settings**, tab **Quality Settings**:
 
 ![Quality settings](images/quality-settings.png)
 
 The following sections explain the settings in detail.
 
-## Commits
+## Gates
 
-These settings configure quality thresholds for commits, as displayed on the [Commits page](../repositories/commits.md).
-
-![Quality settings for commits](images/quality-settings-commits.png)
-
--   **New issues are over:** Commits are marked not up to standards if the number of issues introduced that have at least the specified severity level is bigger than the set value. The default value is 0.
--   **New security issues are over:** Commits are marked not up to standards if the number of security issues introduced is bigger than the set value. By default, this setting is off.
--   **Complexity is over:** Commits are marked not up to standards if the introduced complexity is bigger than the set value. By default, this setting is off.
--   **Duplication is over:** Commits are marked not up to standards if the number of clones introduced is bigger than the set value. By default, this setting is off.
--   **Coverage variation is under:** Commits are marked not up to standards if the coverage introduced is smaller than the set value. This value refers to the variation of coverage on your commit. For example, if your repository has 90% coverage and you want to make sure that this value doesn't decrease, set the value to 0%. By default, this setting is off. The maximum value is 1%.
-
-## Pull Requests
-
-These settings configure quality thresholds for pull requests, as displayed on the Pull Requests page.
+These settings configure which pull requests and commits Codacy reports as up to standards.
```

Comment:
@prcr Should this be "reports as NOT up to standards" ?

Reply:
You're right since the rules actually determine which pull requests and commits "don't pass" the gate. :+1: